rdt/transformers/id.py

```diff
@@ -12,7 +12,7 @@
 LOGGER = logging.getLogger(__name__)
 
 
-class IDGenerator(BaseTransformer):
+class IndexGenerator(BaseTransformer):
```

Comment:
One thing I was unsure on was, if we wanted to also rename the `id.py` file too.

Reply:
Verified with Neha the filename should stay `id.py`